controlpanel/frontend/forms.py

```diff
@@ -235,6 +235,30 @@ def clean_namespace(self):
         return namespace
 
 
+class CloudPlatformArnForm(forms.Form):
+    allow_cloud_platform_assume_role = forms.BooleanField(initial=False, required=False)
+    cloud_platform_role_arn = forms.CharField(
+        required=False, max_length=130, validators=[validators.validate_aws_role_arn]
+    )
+
+    def __init__(self, *args, **kwargs):
+
+        super().__init__(*args, **kwargs)
+
```

Comment:
```suggestion
    def __init__(self, *args, **kwargs):

        super().__init__(*args, **kwargs)

```
No need to define this